README.md

```diff
@@ -17,7 +17,7 @@
 >
 >*"We didn't have a good story about migrating continuations between threads." - Disruptek*
 
-Have you ever asked yourself what would a lock-free MPMC queue look like in nim?
+Have you ever asked yourself what would a lock-free MPMC (*multi-producer multi-consumer*) queue looks like in Nim?
```

Comment:
_look_ is the correct grammatical form here, because we're speaking of the singular -- _a lock-free ... queue_